README.rst

```diff
@@ -532,6 +533,7 @@ References
    models/MITgcm_ocean/trans_pv_sv
    models/MITgcm_ocean/create_ocean_obs
    models/MITgcm_ocean/trans_sv_pv
```

Comment:
```suggestion
   models/MITgcm_ocean/create_ocean_obs
```

Trans_sv_pv and trans_pv_sv don't exist anymore, can `trans_pv_sv.rst` and `trans_sv_pv.rst` be removed?

Reply:
yes